.stickler.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  rubocop:
+    display_cop_names: true
+  csslint:
+  stylelint:
+  eslint:
+    config: './eslint.config'  
```

Comment:
I see you attempted to make this right :clap:. You do still have the wrong configuration however, consider the following [configuration](https://github.com/microverseinc/linters-config/tree/master/css) as well as the steps outlined to enable stickler on your repo.

Reply:
![stickl](https://user-images.githubusercontent.com/12748812/73749980-3478ba00-477e-11ea-946d-3309bcba1b21.png)
I enable the stickler on repo and also add the configuration code from https://github.com/microverseinc/linters-config/tree/master/css here